teste2e/common/k8s.py

```diff
@@ -49,18 +50,26 @@ def __init__(self):
         self.operator_pod_name = run_cmd_text(
             "kubectl get pods | grep mizar-operator | awk '{print $1}'")
 
-    def create_vpc(self, name, ip, prefix, dividers=1):
-        self.api.create_vpc(name, ip, prefix, dividers)
+    def create_vpc(self, name, ip, prefix, dividers=1, vni=None):
+        self.api.create_vpc(name, ip, prefix, dividers, vni)
 
-    def create_net(self, name, ip, prefix, vpc, vni, bouncers=1):
-        self.api.create_net(name, ip, prefix, vpc, vni, bouncers)
+    def create_net(self, name, ip, prefix, vpc, vni, bouncers=1, remote_deployed=False):
+        self.api.create_net(name, ip, prefix, vpc, vni, bouncers, remote_deployed)
 
     def get_vpc(self, name):
         vpc = self.api.get_vpc(name)
         while vpc["status"] != OBJ_STATUS.obj_provisioned:
             vpc = self.api.get_vpc(name)
         return vpc
 
+    def get_vpc_with_status_timeout(self, name, status=OBJ_STATUS.obj_provisioned, timeout=60):
```

Comment:
please rename to 'get_vpc_status_with_timeout'

Reply:
The function is to get a vpc not a vpc status

Reply:
ok. i think the function name makes it sound like it is doing something with timeout. Can you please name it to 'get_vpc_with_status' . The timeout part is fairly intuitive to read from the function args.
Also, if timeout=0, this does not return anything. Is that you intent?

Reply:
I removed the timeout from the function name